examples/index.html

```diff
@@ -86,6 +86,12 @@ <h1><a href="/" target='_blank' class="name-header">Image Sequencer</a></h1>
           </div>
           <p class="info">Select a new module to add to your sequence.</p>
           <div class="row center-align radio-group">
+            <div>
+              <div class="radio" data-value="resize">
+                <i class="icon-resize-full fa-4x i-over"></i>
```

Comment:
You have to add the classes `fa fa-4x i-over` and `fa-name-of-icon`. Do you have any other difficulty?